frontend/src/v5/ui/routes/viewer/drawings/drawingsList/drawingItem/drawingItem.component.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { IDrawing } from '@/v5/store/drawings/drawings.types';
+import { Drawing, Title, DrawingsCalibrationButton, MainBody, ImageContainer, GrayText, BlueText, Description, BottomLine, InfoContainer, BreakingLine } from './drawingtem.styles';
+import { FormattedMessage } from 'react-intl';
+import { formatShortDateTime } from '@/v5/helpers/intl.helper';
+import { DrawingRevisionsHooksSelectors } from '@/v5/services/selectorsHooks';
+
+type DrawingItemProps = {
+	drawing: IDrawing;
+	onClick: React.MouseEventHandler<HTMLDivElement>;
+};
+
+export const DrawingItem = ({ drawing, onClick }: DrawingItemProps) => {
+	const [latestRevision] = DrawingRevisionsHooksSelectors.selectRevisions(drawing._id);
+	const { calibration, name, drawingNumber, lastUpdated, desc } = drawing;
+	const { statusCode, revisionCode } = latestRevision || {};
+
+	return (
+		<Drawing onClick={onClick} key={drawing._id} >
+			<MainBody>
+				<ImageContainer>
+					<img src="https://placedog.net/73/73" />
+				</ImageContainer>
+				<InfoContainer>
+					<BreakingLine>
+						<GrayText>{drawingNumber}</GrayText>
+					</BreakingLine>
+					<BreakingLine>
+						<Title>{name}</Title>
+					</BreakingLine>
+					{statusCode && (
```

Comment:
There are no examples with "status code" in the list.
<img width="363" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/ec928dae-25d7-48d9-bdf1-89d1f16c8c55">